webpack.config.dev.js

```diff
@@ -0,0 +1,22 @@
+const merge = require('webpack-merge');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+let dev = {
```

Comment:
const objecty keys are mutable and some other articles http://softwareengineering.stackexchange.com/questions/278652/how-much-should-i-be-using-let-vs-const-in-es6